CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.25.0] - 2024-07-05
+### ✨ Highlights
+- `pixi exec` command, execute commands in temporary environments, useful for testing in short-lived sessions.
+-  We've bumped the default system-requirements to higher defaults: glibc (2.17 -> 2.28), osx64 (10.15 -> 13.0), osx-arm64 (11.0 -> 13.0). Let us know if this causes any issues.
```

Comment:
Should we mention that:

- to keep the previous defaults for your projects, for example for old RedHat or macOS systems, please overwrite them in the `[system-requirements]` table.

Reply:
Oh yeah good one!

Reply:
Added.